package/MDAnalysis/coordinates/PDBQT.py

```diff
@@ -166,12 +166,13 @@ def _read_first_frame(self):
         self.ts = self._Timestep.from_coordinates(
             coords,
             **self._ts_kwargs)
-        self.ts._unitcell[:] = unitcell
+        self.ts.dimensions = unitcell
```

Comment:
How closely does PDBQT follow the PDB standard? I.e. when reading a 1,1,1,90,90,90 do we need to assign the box dimension to None (probably not to be tackled here, just thinking about whether we need to fix this later).